lib/hangry/parser_selecter.rb

```diff
@@ -21,11 +18,8 @@ def initialize(nokogiri_doc)
     def parser
       # Prefer the more specific parsers
       parser_classes = [
-        Parsers::NonStandard::EatingWellParser,
-        Parsers::NonStandard::EpicuriousParser,
         Parsers::NonStandard::JamieOliverParser,
         Parsers::NonStandard::RachaelRayParser,
-        Parsers::NonStandard::AllRecipesParser
```

Comment:
Just to clarify: are you proposing to ditch these more specific parser classes because they are no longer necessary and/or one of the generic parsers like `JsonLDParser` suffices?

Reply:
Yes! 

You can see how the standard parsers get essentially the same data for
- [eatingwell](https://github.com/lukeasrodgers/hangry/pull/13/files#diff-700f4f7b04d070c5ba8017a211584fd9658c7b894098815c9005699826e91e6f)
- [epicurious](https://github.com/lukeasrodgers/hangry/pull/13/files#diff-700f4f7b04d070c5ba8017a211584fd9658c7b894098815c9005699826e91e6f)

That recipe on allrecipes actually moved to southernliving.com so ... that may not be proof that the allrecipesparser is no longer useful. 🤔